### PR TITLE
Bundle debootstrap with the installer

### PR DESCRIPTION
**Pull request by dnschneid**
Let's bundle the debootstrap scripts with the installer so we don't get hid by surprise issues (except when trying to release).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,16 +49,26 @@ else
...
 	{ \
 		sed -e "s/\$$TARPARAMS/$(TARPARAMS)/" \
 			-e "s/VERSION=.*/VERSION='$(shell $(GENVERSION) $(VERSION))'/" \
 			$(WRAPPER) \
-		&& tar --owner=root --group=root -c $(TARPARAMS) $(SCRIPTS) \
+		&& (cd $(BUILDDIR) && tar --owner=root --group=root -c $(TARPARAMS) *)\
 		&& chmod +x /dev/stdout \
 	;} > $(TARGET) || ! rm -f $(TARGET)
 
+$(BUILDDIR): $(SCRIPTS) Makefile
```
Comment (drinkcat) [inline Makefile]: Missing $(BOOTSTRAPS) as dep?

**Comment by drinkcat** [comment]
LGTM, just one nit.

File: Makefile (hunk 1 above)
Comment (dnschneid) [reply Makefile]: it's included in $(SCRIPTS)

Comment (drinkcat) [reply Makefile]: Ack.

**Comment by dbaynard** [comment]
Hello,

I've been testing an arch linux fork (based on @drinkcat's work!) and this PR makes crouton a 200MB download (as opposed to 100kB). Why should crouton bundle the bootstrap files in its final output, rather than just make sure they actually work?

**Pull request merged**